specify env for biomage rds tunnel (#96)
hms-dbmi-cellenics/biomage-utils: README.md

- Processed RDS file
- Cell sets file

> **Note** this command needs `biomage rds tunnel` running in another tab to work. By default the input environment is staging, so if you want to use production you will need to specify it in the -i option (`biomage rds tunnel -i production`).

Reviewer: ```suggestion
**Note** this command needs `biomage rds tunnel` running in another tab to work. By default, `biomage rds tunnel` connects staging. If you want to use production you need to specify it with the `-i` option (`biomage rds tunnel -i production`).
```
